fvm/fvm.go

```diff
@@ -43,7 +47,12 @@ func (vm *VirtualMachine) GetAccount(ctx Context, address flow.Address, ledger s
 		state.WithMaxValueSizeAllowed(ctx.MaxStateValueSize),
 		state.WithMaxInteractionSizeAllowed(ctx.MaxStateInteractionSize))
 
-	account, err := getAccount(vm, ctx, st, ctx.Chain, address)
+	account, err := getAccount(vm, ctx, st, address)
+	if err != nil {
+		// TODO: wrap error
+		return nil, err
+	}
+	err = st.Commit()
```

Comment:
Is this a bugfix?

Reply:
Not exactly, just moved the state commit to the function that created the state.